objcts.js

```diff
@@ -0,0 +1,9 @@
+for(i=1; i<=15;i++){
+let k = i;
+let j = k*k;
```

Comment:
For this, JavaScript has a new exponentiation operator which we can use to find the square of a number i.e
```js
// instead of
let j = k * k;

// we can use the exponentiation operator instead
let j = k ** 2

// or even use the pow method from Math
let j =  Math.pow(k, 2)
```